src/haddock/modules/topology/topoaa/cns/build-missing.cns

```diff
@@ -170,13 +172,11 @@ if ( $tobuild > 0 ) then
 
         flags exclude * include bond angl impr dihe vdw elec end
 
+        minimize powell nstep=500 nprint=50 end
+
         {- return masses to something sensible -}
         do (mass=refy) (store9)
 
-        do (vx=maxwell($bath)) (store9)
```

Comment:
These are the culprits for the unstable hydrogen positions generation leading to different energies